examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
```

Comment:
What's the point to create a new prototype for each new object? Shuold we just use `descriptor.methods` as the `proto`? 

I was thinking that factory functions is the fast and memory-friendly alternative to Crockford's closures.